cylc/flow/network/schema.py

```diff
@@ -860,6 +860,20 @@ class Meta:
 """Map GQL Runtime fields' names to workflow config setting names."""
 
 
+def runtime_schema_to_cfg(runtime: dict) -> dict:
+    """Covert GQL Runtime field names to workflow config setting names and
+    perform any necessary processing on the values."""
+    # We have to manually lowercase the run_mode field because we don't define
+    # a proper schema for BroadcastSetting (it's just GenericScalar) so
+    # Graphene has no way to know that it should be a TaskRunMode enum.
```

Comment:
FAO @oliver-sanders

Reply:
Is the `graphene.Enum` back to front per-chance?

Reply:
No, the Enum works correctly as far as creating the dropdown and sending the result back

![image](https://github.com/user-attachments/assets/6cd0d176-93e8-4d14-8d73-52caa3697b19)

This is because the Edit Runtime form is built from the result of querying the runtime of the task proxy:
```python
class Runtime(ObjectType):
    ...
    run_mode = TaskRunMode(default_value=TaskRunMode.Live.name)
```
The problem is that in the mutation it sends back, there is no longer any association with this Enum, all you get is `List(BroadcastSetting)` which is the same as `List(GenericScalar)`
```python
class Broadcast(Mutation):
    ...
    class Arguments:
        ...
        settings = graphene.List(
            BroadcastSetting,
            description=sstrip('''
                The `[runtime]` configuration to override with this broadcast
                (e.g. `script = true` or `[environment]ANSWER=42`).
            '''),
```

Reply:
(sorry, was slow on the uptake there)